src/App/Patterns/content/Login/index.js

```diff
@@ -0,0 +1,116 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { DocContainer } from "@docutils";
+import { ComponentPreview } from "~/src/App/docutils";
+
+import InputGroup from "@components/InputGroup";
+import LoginForm from "../../components/LoginForm/index";
+
+import { guidelineList } from "./constants";
+
+const basename = process.env.basename;
+
+const Overview = () => (
+    <section>
+        <h2 id="overview">Overview</h2>
+
+        <img src={`${basename}img/swedbankpay/documentation/patterns/login-pattern-example.png`} className="d-none d-md-block w-100 border-left border-right border-bottom" alt="Image for login screen" />
```

Comment:
This path doesn't exist when project is built:
`${basename}img/swedbankpay/documentation/patterns/login-pattern-example.png`

The culprit here is the brand. We might have to copy these assets to a brand folder in webpack.

Reply:
I believe we fixed this